src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1610,8 +1613,10 @@ class ApplicationManagerImpl
     mobile_apis::SystemContext::eType system_context;
   };
 
-  hmi_apis::HMI_API* hmi_so_factory_;
-  mobile_apis::MOBILE_API* mobile_so_factory_;
+  hmi_apis::HMI_API hmi_so_factory_;
+  mobile_apis::MOBILE_API mobile_so_factory_;
+  ns_smart_device_link_rpc::V1::v4_protocol_v1_2_no_extra
+      v4_protocol_so_factory_;
```

Comment:
I think this variable name should include `mobile` to match the other api schemas and make it more obvious what this schema is.

Reply:
Done in 4c89818